### PR TITLE
fix: Spacing not available in rtl view

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.

closes: https://github.com/SAP/fundamental-ngx/issues/1268

#### Please provide a brief summary of this pull request.

when we are using the icon based select component in the rtl view there is no spacing between the icon and the placeholder so introduced a class to accomodate this change

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples

### CODE REVIEW

**Comment by JKMarkowski** [comment]
Hi @Lokanathan-k, can you put it to `.scss` file? Let's not use inline-styling anymore.

File: libs/core/src/lib/select/select.component.scss [hunk 1]
```diff
@@ -108,6 +108,9 @@
         pointer-events: none;
         margin-right: auto;
     }
```
Comment (JKMarkowski) [inline libs/core/src/lib/select/select.component.scss]: Let's just add new line between classes, then we can merge it.

File: libs/core/src/lib/select/select.component.scss [hunk 2]
```diff
@@ -108,6 +108,10 @@
...
+	
+    .icon-position{
+        margin: 1px 12px
+	}
```
Comment (fkolar) [inline libs/core/src/lib/select/select.component.scss]: missing colon. We shoudl add more restrictive lint rules.

**Comment by mikerodonnell89** [comment]
The RTL toggle for the one with icons actually toggles the previous example

**Comment by mikerodonnell89** [comment]
It's not a result of code changes in this PR, but I noticed the RTL toggle for the second example actually toggles the RTL for the first, due to both examples using `[name]="'ex1'"`

**Comment by droshev** [comment]
@Lokanathan-k do you need more time for the PR? It is getting a bit old.
Thank you!

**Comment by Lokanathan-k** [comment]
> @Lokanathan-k do you need more time for the PR? It is getting a bit old.
> Thank you!

@droshev  i dont have any code change need approval from @fkolar post that i can merge the change to the master till then i am unable to merge the code

**Comment by droshev** [comment]
@fkolar do you think you can take a look?

File: libs/core/src/lib/select/select.component.scss (hunk 2 above)
Comment (Lokanathan-k) [reply libs/core/src/lib/select/select.component.scss]: fixed pls review @fkolar

**Pull request merged**